README.md

```diff
@@ -37,7 +37,9 @@ It will compile a project for the `F4` family by default, but you can also compi
 # Usage
 
 First of all you need to configure toolchain and library paths using CMake variables. 
```

Comment:
Maybe we could add the variables names (or an example of variable name) here ?

Reply:
I think this is a good idea. maybe even a concrete example for a file which can be sourced to load all necessary env variables at once (I find this really convenient compared to very large CMake commands, and the path stays clean if it is not used)

Reply:
As of commit 08ceb4b it seems already excellent to me. It's better than what I expected. I will review When you remove WIP.

Please note also there are also Windows users like me, so keep generic (even if we can consider Linux as default choice)

Reply:
I actually used `.sh` scripts with git bash or MinGW64. I am not totally sure whether this belongs in the README, but a regular CMake workflow generally involves more scripting and I found scripts like that to set up environments quickly extremely useful. I added this for the powershell on Windows as well, but I need to test this. I read somewhere that it would not be possible with powershell like that

Reply:
The .sh only is OK don't worry. I just wanted to heads up. Your work is great.